CMakeLists.txt

```diff
@@ -148,19 +153,28 @@ if(NOT HAVE_SYS_UIO_H_01)
   set(HAVE_SYS_UIO_H_01 0)
 endif(NOT HAVE_SYS_UIO_H_01)
 
-if (SNAPPY_FUZZING_BUILD)
-  if (NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+if(SNAPPY_FUZZING_BUILD OR SNAPPY_OSSFUZZ_BUILD)
+  if(NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
     message(WARNING "Fuzzing builds are only supported with Clang")
-  endif (NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+  endif(NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
```

Comment:
Warn if either the CI fuzzing build or the ossfuzz build is not being compiled via clang